srcpkgs/qpwgraph/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'qpwgraph'
+pkgname=qpwgraph
+version=0.0.2
+revision=1
+wrksrc="${pkgname}-v${version}"
+build_style=cmake
+hostmakedepends="pkg-config qt6-base"
+makedepends="qt6-base-devel pipewire-devel"
```

Comment:
Any reason to prefer qt6 over qt5?

Reply:
not really, I would just stick to what the developer uses more often, which I think is qt6. But if there are reasons to prefer qt5, it's ok.